### PR TITLE
Product Roadmap in Documentation

### PR DESCRIPTION
**Pull request by Tanmay-97**
Reviewer: @sidravi1 

Estimate: 15m

---

## Ticket

Fixes: [AAQ-305](https://idinsight.atlassian.net/browse/AAQ-305)

## Description, Motivation and Context
A rough roadmap should be there in the docs so it's easily share-able externally.

### Goal
Add a new page in MKDocs documentation that has a table

### Changes
-
## How has this been tested?
Locally

## Checklist

Fill with `x` for completed. Delete any lines that are not relevant

- [x ] My code follows the style guidelines of this project
- [x ] I have reviewed my own code to ensure good quality
- [x ] I have tested the functionality of my code to ensure it works as intended
- [ x] I have resolved merge conflicts
- [ ] I have updated the automated tests (if applicable)
- [ ] I have updated the requirements (if applicable)
- [ ] I have updated the README file (if applicable)
- [ ] I have updated affected documentation (if applicable)

[AAQ-305]: https://idinsight.atlassian.net/browse/AAQ-305?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Tanmay-97** [comment]
Preview
image

File: docs/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+| Q3 2024     | Content Management Dashboard & Engineering Dashboard <br><br><br> Multi-turn chat <br> <br><br> Document-based question answering | :pencil: <br><br><br><br><br>:pencil: <br><br><br>:pencil:         | Monitor uptime, response rates, throughput HTTP response codes. See content use, questions that receive poor feedback, missing content, and more. <br><br> Refine or clarify user question through conversation. <br><br> Retrieve answers from a document set |
+| Q4 2024     | Voice notes support <br><br><br> Multimedia content <br><br><br> A/B Testing                                                      | :pencil: <br><br><br>:pencil: <br><br><br>:pencil:                 | Automatic Speech Recognition for audio message to content matching <br><br> Respond with not just text but images and audio as well. <br> <br> Test and decide content that works better for users                                                             |
+
+!!! info "Key <br> :white_check_mark: - Completed <br> :tools: - In active development <br> :construction: - Queued <br>:pencil: - Yet to be scoped"
+
+On The Horizon
```
Comment (sidravi1) [inline docs/roadmap.md:11]: How should I think about "On the Horizon" vs. the ones above?

**Review by sidravi1 — APPROVED** [review]
Looks good @Tanmay-97! Thank you.
What does "On the Horizon" imply? Beyond 2024 or "any day now"? That's my only comment. Approving!

File: docs/roadmap.md (hunk 1 above)
Comment (Tanmay-97) [reply docs/roadmap.md:11]: Thanks @sidravi1, will make it `Beyond 2024` then

**Pull request merged**